docs/OV_Runtime_UG/ShapeInference.md

```diff
@@ -267,21 +269,21 @@ For details, refer to the [Tensorflow Object Detection API models resizing techn
 
 Some operators which prevent normal shape propagation can be fixed. To do so you can:
 * see if the issue can be fixed via changing the values of some operators input. 
-E.g. most common problem of non-reshape-able models is a `Reshape` operator with hardcoded output shape.
+For example, most common problem of non-reshape-able models is a `Reshape` operator with hardcoded output shape.
 You can cut-off hard-coded 2nd input of `Reshape` and fill it in with relaxed values.
 For the following example on the picture Model Optimizer CLI should be:
 ```sh
 mo --input_model path/to/model --input data[8,3,224,224],1:reshaped[2]->[0 -1]`
 ```
-With `1:reshaped[2]` we request to cut 2nd input (counting from zero, so `1:` means 2nd inputs) of operation named `reshaped` and replace it with a `Parameter` with shape `[2]`.
-With `->[0 -1]` we replace this new `Parameter` by a `Constant` operator which has value `[0, -1]`. 
-Since `Reshape` operator has `0` and `-1` as a specific values (see the meaning in [the specification](../ops/shape/Reshape_1.md)) it allows to propagate shapes freely without losing the intended meaning of `Reshape`.
+With `1:reshaped[2]` you request to cut 2nd input (counting from zero, so `1:` means 2nd inputs) of operation named `reshaped` and replace it with a `Parameter` with shape `[2]`.
+With `->[0 -1]` you replace this new `Parameter` by a `Constant` operator which has value `[0, -1]`. 
+Since `Reshape` operator has `0` and `-1` as a specific values (see the meaning in [the specification](../ops/shape/Reshape_1.md)) it allows propagating shapes freely without losing the intended meaning of `Reshape`.
```

Comment:
```suggestion
Since the `Reshape` operator has `0` and `-1` as specific values (see the meaning in [this specification](../ops/shape/Reshape_1.md)), it allows propagating shapes freely without losing the intended meaning of `Reshape`.
```